Source/Tx.Network/UDPReceiver.cs

```diff
@@ -159,6 +159,14 @@ private void Dispose(bool disposing)
                 disposeCalled = true;
                 Socket.Shutdown(SocketShutdown.Both);
                 Socket.Dispose();
+                
+                while(_receivedDataProcessorsPool.Count > 0)
+                {
+                    var dequeued = new SocketAsyncEventArgs();
```

Comment:
This should just be a declaration of a variable of type ```SocketAsyncEventArgs``` because the parameter to the ```TryDequeue``` method is an out parameter.